stable/openstack-telemetry/README.md

```diff
@@ -1,9 +1,308 @@
-# OpenStack Telemetry
+# Basic OpenStack cloud
 
-This example bundle extends the basic OpenStack Cloud bundle with Telemetry collection via Ceilometer.
+The `openstack-telemetry` bundle currently deploys a basic OpenStack cloud with
+these foundational elements:
+
+- Ubuntu 20.04 LTS (Focal)
+- OpenStack Ussuri
+- Ceph Octopus
+
+## Requirements
+This example bundle is designed to run on bare metal using Juju 2.x with [MAAS][] (Metal-as-a-Service); you will need to have setup a [MAAS][] deployment with a minimum of 4 physical servers prior to using this bundle.
 
 Certain configuration options within the bundle may need to be adjusted prior to deployment to fit your particular set of hardware. For example, network device names and block device names can vary, and passwords should be yours.
 
-For full details on the base cloud deployment please refer to the [Basic OpenStack Cloud][] bundle.
+For example, a section similar to this exists in the bundle.yaml file.  The third "column" are the values to set.  Some servers may not have eno2, they may have something like eth2 or some other network device name.  This needs to be adjusted prior to deployment.  The same principle holds for osd-devices.  The third column is a whitelist of devices to use for Ceph OSDs.  Adjust accordingly by editing bundle.yaml before deployment.
+
+Servers should have:
+
+ - A minimum of 8GB of physical RAM.
+ - Enough CPU cores to support your capacity requirements.
+ - Two disks (identified by /dev/sda and /dev/sdb); the first is used by MAAS for the OS install, the second for Ceph storage.
+ - Two cabled network ports on eno1 and eno2 (see below).
+
+Servers should have two physical network ports cabled; the first is used for general communication between services in the Cloud, the second is used for 'public' network traffic to and from instances (North/South traffic) running within the Cloud.
```

Comment:
It would be useful for newcomers to have a reference to this appendix:
https://docs.openstack.org/project-deploy-guide/charm-deployment-guide/latest/app-bridge-interface-configuration.html

In my experience, this is by far the most confusing part for people attempting to deploy OpenStack for the first time.